lib/util/got/renovate-agent.js

```diff
@@ -0,0 +1,9 @@
+const got = require('got');
+
+// Sets the user agent to be Renovate
+
+module.exports = got.extend({
+  headers:
+    process.env.RENOVATE_USER_AGENT ||
+    'https://github.com/renovatebot/renovate',
+});
```

Comment:
`headers` must be an object. The value should be wrapped in `user-agent` key. Also, it's a good thing to expose `package.version`.

```diff
const got = require('got');
+const package = require('../../../package.json');

// Sets the user agent to be Renovate

module.exports = got.extend({
-  headers:
-    process.env.RENOVATE_USER_AGENT ||
-    'https://github.com/renovatebot/renovate',
+  headers: {
+    'user-agent': process.env.RENOVATE_USER_AGENT || `renovate/${package.version} (https://github.com/renovatebot/renovate)`
+  }
});
```

Reply:
Oops, that was a typo. now fixed